plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list/payment-gateway-list.tsx

```diff
@@ -0,0 +1,126 @@
+/**
+ * External dependencies
+ */
+import { PaymentProvider, PaymentProviderType } from '@woocommerce/data';
+import { getAdminLink } from '@woocommerce/settings';
+import { Gridicon } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import {
+	DefaultDragHandle,
+	SortableContainer,
+	SortableItem,
+} from '~/settings-payments/components/sortable';
+import { PaymentExtensionSuggestionListItem } from '~/settings-payments/components/payment-extension-suggestion-list-item';
+import { PaymentGatewayListItem } from '~/settings-payments/components/payment-gateway-list-item';
+import './payment-gateway-list.scss';
+
+interface PaymentGatewayListProps {
```

Comment:
Nitpick: The type interface could be relocated to `payment-settings/types.ts`.

Reply:
Thanks for the suggestion! Because the props are only used in this one place, I preferred keeping them in the same file - but in the future we can probably extract a lot of the shared gateway logic and then use a shared types file as you suggest. I will keep it in mind :)